install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -129,18 +129,50 @@ It is recommended to deploy both groups to avoid potential impacts on
 performance in I/O and volume creation. Both of these are defined in the
 inventory hosts file.
 
-The definition of the clusters is done by including the relevant names in the
-`[OSEv3:children]` group, creating similarly named groups, and then populating
-the groups with the node information. The clusters can then be configured
-through a variety of variables in the `[OSEv3:vars]` group. `glusterfs`
-variables begin with `openshift_storage_glusterfs_` and `glusterfs_registry`
-variables begin with `openshift_storage_glusterfs_registry_`. A few other
-variables, such as `openshift_hosted_registry_storage_kind`, interact with the
-GlusterFS clusters.
-
-To prevent {gluster} pods from upgrading after an outage leading to a cluster
-with different {gluster} versions, it is recommended to specify the image name
-and version tags for all containerized components. The relevant variables are:
+To define the storage clusters, include the relevant names in the
+`[OSEv3:children]` group, creating similarly named groups. Then populate
+the groups with the node information.
+
+After the groups are created and populated, you can then configure the clusters
+using a variety of variables in the `[OSEv3:vars]` group. The variables interact
+with the GlusterFS clusters.
+
+- `glusterfs` variables begin with `openshift_storage_glusterfs_`.
```

Comment:
Should this be in the next section?

Reply:
I don't think so. We've been talking about variables, and this gives more contextual information. I think it goes well here.

Reply:
If you refer the link provided you will get more information about this from the inventory file.

https://docs.google.com/document/d/1-WyKSDz4xPNLEYYrptbW3G9jAZDAlRH61WusYr3-e9c/edit?usp=sharing

Reply:
Included the example.